app/assets/javascripts/fe/fe.public.nojquery.js

```diff
@@ -344,6 +344,8 @@
         vals = $([$element.find("input[type=radio]:checked").val()]);
       } else if ($element.hasClass('fe_choicefield') && $element.hasClass('checkbox')) {
         vals = $element.find("input[type=checkbox]:checked").map(function(i, el) { return $(el).val(); });
```

Comment:
this one was necessary because otherwise it was returning the first checked value only.. I want to match any checked off option with any of the conditional answers